README.md

```diff
@@ -263,6 +263,34 @@ JAEGER_AGENT_HOST: "localhost"
 JAEGER_AGENT_PORT: 6831
 ```
 
+## Stream Joins
+This will allow an actor to join messages from 2 topics into 1 result. To be able to use stream joins just add the configuration below to your `config.edn`
+```clojure
+{:ziggurat  {:stream-router        {:stream-id            {
+    :consumer-type        :stream-joins
+    :input-topics         {:topic-1 {} :topic-2 {}}
+    :join-cfg             {:topic-1-and-topic-2 {:join-window-ms 5000 :join-type :inner}}
```

Comment:
if :consumer-type is not specified, will it automatically start running as a normal consumer?

Reply:
done